### ISSUE TITLE
WCAG: Assess application using automated tools

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/story-ramp#24]
A WCAG report on the state of compliance is updated to reflect results from an automated assessment of the application.

**Comment by dan-bowerman** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/StoryRAMP-61810f925832ab001db602b8/issues/ramp4-pcar4/story-ramp/24?planning-poker) with ZenHub @JahidAhmed @RyanCoulsonCA @yileifeng

**Comment by dan-bowerman** [comment]
Use the [WAVE toolbar](https://wave.webaim.org/extension/) to assess if there are any obvious problems with the application. If there are issues, create a new issue with a list of things that need to be addressed. Any task that seems huge should be in its own issue.

**Comment by spencerwahl** [comment]
Errors (need to be fixed):
- `Skip to main content` link doesn't work. This is only accessible by keyboard by tabbing at the top of the page.
- Links in the `Context` footer are low-contrast. The purple visited links are fine but the blue needs to be darkened.

Possible issues:
- Some images have long alternate text. From what I could tell the images are pretty complicated where this pops up so not sure if there is much to do.
- A header level is skipped in panel 4 of Oil sands. The sub-header should be h3 instead of h4 (this help screen readers)
- A header level is skipped in panel 3 of Ethylene Glycol. Same issue as above, probably a quick fix in changing the component.

Otherwise things seem to be pointing to invisible elements or various things within RAMP instances (empty headers, small text, etc.)
